### PR TITLE
Refactoring KeysetPaginator

### PR DESCRIPTION
**Pull request by romkatsu**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ✔️
| Tests pass?   | ✔️
| Fixed issues  | #35

### CODE REVIEW

File: src/Paginator/KeysetPaginator.php [hunk 1]
```diff
@@ -27,41 +29,40 @@ class KeysetPaginator implements PaginatorInterface
      * @var FilterableDataInterface|DataReaderInterface|SortableDataInterface
      */
     private $dataReader;
```
Comment (roxblnfk) [inline src/Paginator/KeysetPaginator.php:31]:
```suggestion
    private DataReaderInterface $dataReader;
```

Comment (romkatsu) [reply src/Paginator/KeysetPaginator.php]: fixed

**Comment by romkatsu** [comment]
[Refactoring KeysetPaginator](https://trello.com/c/vT5vrFg0/62-refactoring-keysetpaginator)

File: src/Paginator/KeysetPaginator.php (hunk 1 above)
Comment (xepozz) [reply src/Paginator/KeysetPaginator.php]: No, it must be union of `FilterableDataInterface|DataReaderInterface|SortableDataInterface`

Comment (roxblnfk) [reply src/Paginator/KeysetPaginator.php]: > No, it must be union of `FilterableDataInterface|DataReaderInterface|SortableDataInterface`

and this placed in the annoation

**Comment by samdark** [comment]
👍

**Pull request merged**